### DISCUSSION TITLE
MainButton shows but doesnt work/do anything when clicked on Mobile App but it works on web version of telegram

### DISCUSSION THREAD

**Discussion by kali-kb** [vkruglikov/react-telegram-web-app#69] [Q&A]
Hello, after i initialized the core library <script src="https://telegram.org/js/telegram-web-app.js"></script> in index.html, after following the docs carefully, i couldnt get the MainButton to work in my mini-app from Telegram in my Android device, the button appears but when i click on it it doesnt do anything, but it works on the web version of telegram here is the code that i used

import { MainButton, useShowPopup } from '@vkruglikov/react-telegram-web-app';

```
import { MainButton, useShowPopup } from '@vkruglikov/react-telegram-web-app';
const Content = () => {
  const showPopup = useShowPopup();

  const handleClick = () =>
    showPopup({
      message: 'Hello, I am popup',
    });

  return <MainButton text="SHOW POPUP" onClick={handleClick} />;
};
```
i took this from example the document provides, this is the code that isnt working for me, am i missing something?, thanks in advance

**Comment by TrickyXcs** [comment]
Same problem. Maybe found a way to fix it?

**Comment by megz2020** [comment]
same problem
